### PR TITLE
Creation of a main file to get the ball rolling

### PR DESCRIPTION
**Pull request by rogerhurwitz**

### CODE REVIEW

File: pyvault.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"""
+PyVault is a text-based password management application
+"""
+def main():
+   pass
+
+if __name__ == "__main__":
+    main()
```
Comment (jhurwitzupenn) [inline pyvault.py]: Please add newline at end of file

File: pyvault.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+"""
```
Comment (jhurwitzupenn) [inline pyvault.py:1]: Run yapf to format your code.

**Review by jhurwitzupenn — CHANGES_REQUESTED** [review]
Mainly formatting changes.

**Comment by rogerhurwitz** [comment]
Used PyCharm reformatter, not yapf, to reformat file.  It corrected the newline issue.

**Review by jhurwitzupenn — APPROVED** [review]

**Pull request merged**